UI.tsv

```diff
@@ -1054,16 +1054,16 @@ UI_20150317_001053	Characters Water Reflection
 UI_20150317_001054	Background Water Reflection
 UI_20150317_001055	{s20}{#BBBBBB}Glow Setup
 UI_20150317_001056	Use Glow
-UI_20150317_001057	{@st41}ATK{/}
-UI_20150317_001058	{@St41}DEF{/}
-UI_20150317_001059	{@st41} AUX{/}
-UI_20150317_001060	{@St41}          Difficulty{/}
+UI_20150317_001057	{@st41}Attack{/}
+UI_20150317_001058	{@St41}Defense{/}
+UI_20150317_001059	{@st41}Support{/}
+UI_20150317_001060	{@St41}Difficulty{/}
 UI_20150317_001061	{@St41}Hair
```

Comment:
Hairstyle sounds better than Hair yeah, I'll add this extra edit in the re-submitted pull request.